libs/SalesforceReact/src/com/salesforce/androidsdk/reactnative/ui/SalesforceReactActivity.java

```diff
@@ -100,6 +103,9 @@ public void onDestroy() {
 
     @Override
     public void onResume() {
+        if(client != null && reactActivityDelegate != null){
```

Comment:
This call should come after `super.onResume()`. Also, I'm not sure if this will work for first time load, because `client` will be `null` the first time, since the `peekRestClient()` call is below and that's what initiates login flow.